dimod/constrained.py

```diff
@@ -493,7 +530,7 @@ def add_discrete_from_comparison(self,
         Args:
             comp: Comparison object. The comparison must be a linear
                 equality constraint with all of the linear biases on the
-                left-hand side equal to one and the right hand side equal
+                left-hand-side equal to one and the right-hand-side equal
```

Comment:
```suggestion
                left-hand side equal to one and the right-hand side equal
```

Reply:
I see, I had copied your double hyphen from https://github.com/dwavesystems/dimod/blame/main/docs/intro/intro_symbolic_math.rst#L163, but I can change them all to single.

Reply:
> I see, I had copied your double hyphen from https://github.com/dwavesystems/dimod/blame/main/docs/intro/intro_symbolic_math.rst#L163, but I can change them all to single.

Yes, the "left-hand side(``lhs``)" is correct but the ", right-hand-side" has an extra hyphen there